src/cdk/menu/menu-trigger.ts

```diff
@@ -191,20 +192,20 @@ export class CdkMenuTrigger extends CdkMenuTriggerBase implements OnDestroy {
    * into.
    */
   private _subscribeToMouseEnter() {
-    // Closes any sibling menu items and opens the menu associated with this trigger.
-    const toggleMenus = () =>
-      this._ngZone.run(() => {
-        this._closeSiblingTriggers();
-        this.open();
-      });
-
     this._ngZone.runOutsideAngular(() => {
       fromEvent(this._elementRef.nativeElement, 'mouseenter')
         .pipe(
           filter(() => !this.menuStack.isEmpty() && !this.isOpen()),
           takeUntil(this.destroyed),
         )
         .subscribe(() => {
+          // Closes any sibling menu items and opens the menu associated with this trigger.
+          const toggleMenus = () =>
```

Comment:
Unrelated change, I just noticed that we can move this function closer to its call site.